sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -0,0 +1,291 @@
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.AreEqual(documents.Count, resultCollection.Statistics.DocumentCount);
+
+            Assert.AreEqual(48, resultCollection[0].Statistics.Value.CharacterCount);
+            Assert.AreEqual(1, resultCollection[0].Statistics.Value.TransactionCount);
+
+            Assert.Greater(resultCollection.Statistics.DocumentCount, 0);
+            Assert.AreEqual(2, resultCollection.Statistics.DocumentCount);
+            Assert.AreEqual(2, resultCollection.Statistics.TransactionCount);
+            Assert.AreEqual(0, resultCollection.Statistics.InvalidDocumentCount);
+            Assert.AreEqual(2, resultCollection.Statistics.ValidDocumentCount);
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesBatchTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            List<TextDocumentInput> documents = batchDocuments;
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(documents);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.GreaterOrEqual(resultCollection.Count, 2);
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesBatchWithStatisticsTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = batchDocuments;
+
+            HealthcareOptions options = new HealthcareOptions()
+            {
+                IncludeStatistics = true
+            };
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(documents, options);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.GreaterOrEqual(resultCollection.Count, 2);
+
+            Assert.Greater(resultCollection.Statistics.DocumentCount, 0);
+            Assert.AreEqual(2, resultCollection.Statistics.DocumentCount);
+            Assert.AreEqual(2, resultCollection.Statistics.TransactionCount);
+            Assert.AreEqual(0, resultCollection.Statistics.InvalidDocumentCount);
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesBatchWithPagination()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = @"RECORD #333582770390100 | MH | 85986313 | | 054351 | 2/14/2001 12:00:00 AM | CORONARY ARTERY DISEASE.";
+
+            var list = new List<string>();
+
+            for (int i = 0; i < 23; i++)
+            {
+                list.Add(document);
+            };
+
+            HealthcareOperation healthOperation = await client.StartHealthcareBatchAsync(list);
+
+            AsyncPageable<DocumentHealthcareResult> results = client.GetHealthcareEntities(healthOperation);
```

Comment:
hmm yeah, what happens if the user does:
```
HealthcareOperation operation = await client.StartHealthcareBatchAsync(documents, options);
await operation.WaitForCompletionAsync();
RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
```
I think this is confusing. Better to integrate it into the HealthcareOperation

Reply:
Let's leave this for after preview.1